pyproject.toml

```diff
@@ -55,18 +55,48 @@ branch = true
 source = ["src"]
 command_line = "-m pytest"
 
-[tool.isort]
-lines_after_imports = 2
-force_single_line = true
-no_lines_before = ["FUTURE", "STDLIB", "THIRDPARTY", "FIRSTPARTY", "LOCALFOLDER"]
-known_first_party = ["nplinker"]
-src_paths = ["src", "tests"]
-line_length = 80
-
 # Mypy: to check static type
 # see https://mypy.readthedocs.io/en/stable/config_file.html#example-pyproject-toml
 [tool.mypy]
 python_version = "3.10"
 warn_return_any = true
 warn_unused_configs = true
 ignore_missing_imports = true
+
+[tool.ruff]
```

Comment:
What is the purpose of having this section in the toml file?

Reply:
It's the configurations of ruff (see [here1](https://docs.astral.sh/ruff/configuration/) and [here2](https://docs.astral.sh/ruff/settings/)). If not configuring it, ruff will use its [default values](https://docs.astral.sh/ruff/settings/) that might not be what we need on code styles, for example the [maximum of line length](https://docs.astral.sh/ruff/settings/#line-length) is 88 by default but we want it to be 100.

Our [python template](https://github.com/NLeSC/python-template) also adopted ruff, you could take a look there too.